localstack-core/localstack/config.py

```diff
@@ -742,6 +742,11 @@ def populate_edge_configuration(
 # Whether to enable the partition adjustment listener (in order to support other partitions that the default)
 ARN_PARTITION_REWRITING = is_env_true("ARN_PARTITION_REWRITING")
 
+# Feature flag to enable validation of internal endpoint responses in the handler chain. For test use only.
+OPENAPI_VALIDATE_RESPONSE = is_env_true("OPENAPI_VALIDATE_RESPONSE")
+# Flag to enable the validation of the requests made to the LocalStack internal endpoints. Active by default.
+OPENAPI_VALIDATE_REQUEST = is_env_not_false("OPENAPI_VALIDATE_REQUEST")
```

Comment:
q: Should this config maybe be enabled by default in our test pipeline?

Reply:
Shouldn't be the case already with `is_env_not_false`?

Reply:
Sorry, that's true, jumped a few lines here. I meant `OPENAPI_VALIDATE_RESPONSE`.